03-components/50-weather-components/WeatherApp.js

```diff
@@ -1,52 +1,30 @@
 import { defineComponent } from 'vue'
-import { getWeatherData, WeatherConditionIcons } from './weather.service.ts'
 import './WeatherApp.css'
+import { getWeatherData, WeatherConditionIcons } from './weather.service.ts'
+import WeatherList from './components/WeatherList.js'
 
 export default defineComponent({
   name: 'WeatherApp',
 
+  components: {
+    WeatherList,
+  },
+
+  setup() {
+    const cities = getWeatherData()
+
+    const weatherIcons = WeatherConditionIcons
```

Comment:
These icons are good in the components where they are actually used. It's unlikely that a user will use these component with different icons set. Especially if a user **have** to provide these icons, and there is no default value.

And even more if it's a part of interface of all the intermediate components like `WeatherList`.